assets/components/lvms/topolvm-node-metrics_v1_service.yaml

```diff
@@ -14,6 +14,6 @@ spec:
     protocol: TCP
     targetPort: 8080
   selector:
-    app.kubernetes.io/name: topolvm-node
+    app.kubernetes.io/component: topolvm-node
```

Comment:
Sorry for digging this out, but it got me thinking...

We updated this file as it's not generated in runtime by the operator, but we're missing other part of the equation - labels in daemonset.
Looking at https://github.com/red-hat-storage/lvm-operator/blob/08b37d20e0711b72dabf884eeb77b228cde7c7fe/controllers/topolvm_node.go#L178-L182 it looks we also need to update labels in our topolvm-node_daemonset.yaml. 
Currently, we create Service, but it has nothing to point to?

Reply:
While we wait for lvms team to provide us yamls, I think it could be faster to ask them to export ` getNodeDaemonSet()` and other functions so we could introduce yet another language to our rebase automation and generate these manifests ourselves :see_no_evil:

Reply:
As suggested, I generated the deployment and daemonset manifests (the only 2 not available in the operator bundle) and selectively merged them onto our current manifests.  I ignored upstream values for resource requests, since MicroShift runs with reduced values. Also ignored image refs since inject those those via go template.  Lastly, I left off the snapshotter container spec.

Reply:
> While we wait for lvms team to provide us yamls, I think it could be faster to ask them to export getNodeDaemonSet() and other functions so we could introduce yet another language to our rebase automation and generate these manifests ourselves 🙈

I think taking it one step further and adding a "render" command to the operator would be easier to manager from our perspective.